### PR TITLE
Add raw example type to contexture client

### PR DESCRIPTION
**Pull request by stellarhoof**
Follow up to https://github.com/smartprocure/contexture/pull/234

### CODE REVIEW

**Review by JBezerra — APPROVED** [review]

File: packages/client/src/exampleTypes.js [hunk 1]
```diff
@@ -443,4 +443,9 @@ export default F.stampKey('type', {
       drilldown: 'self',
     },
   },
+  raw: {
+    reactors: {
+      filter: 'self',
```
Comment (daedalus28) [inline packages/client/src/exampleTypes.js:448]: This should probably be others, not self.

Comment (stellarhoof) [reply packages/client/src/exampleTypes.js:448]: Oh hey welcome! :)

How so? For context, we're using a raw node for a [TermsStatsTable](https://github.com/smartprocure/contexture/blob/main/packages/react/src/exampleTypes/TermsStatsTable.js#L66) and without the client reacting to changes to `filter` the table never refreshes.

Comment (daedalus28) [reply packages/client/src/exampleTypes.js:448]: `filter` on a `raw` node affects other nodes in the tree by changing its filter. A self reactor tells it to update its own context (rerunning the result function). 

In your example, updating the raw node should update the terms stats node, not itself.

Comment (daedalus28) [reply packages/client/src/exampleTypes.js:448]: To further clarify - if you’re using `raw` and expecting it’s `filter` property to work like terms stats, you’re likely to be surprised that it’s affecting the entire rest of the search and not just its own context.

**Pull request merged**